virttest/utils_test/libvirt.py

```diff
@@ -3899,6 +3899,11 @@ def check_qemu_cmd_line(content,
         cmd_result = remote_old.run_remote_cmd(cmd, remote_params, runner_on_target)
         qemu_line = cmd_result.stdout
     search_result = True if re.search(r'%s' % content, qemu_line) else False
```

Comment:
It seems this line should be removed, right?

Reply:
Yes ,DONE